### PR TITLE
Sp4 357 fix using env in ios appdelegate file fe

### PR DESCRIPTION
**Pull request by serhiibets**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -131,3 +131,4 @@ app.*.symbols
 
 wayat/.env
 SPIKES/firestore/firestore_example/android/app/google-services.json
+wayat/frontend/ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme
```
Comment (capf4bbo) [inline .gitignore:134]: This line is not necessary

File: wayat/frontend/pubspec.yaml [hunk 2]
```diff
@@ -65,6 +65,7 @@ dependencies:
   image_picker: ^0.8.5+3
   # Testing mocks
   mockito: ^5.3.0
+  smart_auth: ^1.0.5
```
Comment (capf4bbo) [inline wayat/frontend/pubspec.yaml:68]: Is this necessary?

**Review by capf4bbo — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (serhiibets) [reply .gitignore:134]: Ok

**Review by capf4bbo — APPROVED** [review]
LGTM

**Pull request merged**